activerecord/test/cases/adapters/postgresql/range_test.rb

```diff
@@ -302,9 +302,12 @@ def test_escaped_tsrange
   end
 
   def test_unbounded_tsrange
-    tz = ::ActiveRecord.default_timezone
-    assert_equal_round_trip @first_range, :ts_range, Time.public_send(tz, 2010, 1, 1, 14, 30, 0)...nil
-    assert_equal_round_trip @first_range, :ts_range, nil..Time.public_send(tz, 2010, 1, 1, 14, 30, 0)
+    time = Time.public_send(::ActiveRecord.default_timezone, 2010, 1, 1, 14, 30, 0)
+
+    assert_equal_round_trip @first_range, :ts_range, time..nil
```

Comment:
This is the key assertion. I'm assuming it was never added because the previous committer kept experiencing it being flipped from `..` to `...`.